samples/snippets/csharp/VS_Snippets_CLR/Asynchronous_File_IO_async/cs/AsynchronousFileIO.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0-windows</TargetFramework>
+    <Nullable>enable</Nullable>
+    <UseWPF>true</UseWPF>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
```

Comment:
Why is this needed?

Reply:
@Youssef1313  Because we define the same MainWindow class in example.cs and example3.cs and example4.cs

Reply:
The problem with `EnableDefaultCompileItems` being false is that it beats the purpose of the csproj, which is to make sure the code compiles properly. If the examples are completely independent, I'd make a folder for each of them, each with its own csproj. @IEvangelist Do you agree?

Reply:
Actually `example3` and `example4` doesn't seem to be referenced. So they can just be deleted?

Reply:
@IEvangelist Can I delete the example3.cs and example4.cs file and create the MainWindow2.xaml file?

Reply:
You'd have to verify that those two files are in fact not being referenced, if you can prove that then yes.